packages/admin-ui/src/HeaderBar/HeaderBar.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { makeDecoratable } from "~/utils";
+import { Separator } from "~/Separator";
+
+interface HeaderBarProps {
+    left?: React.ReactNode;
+    right?: React.ReactNode;
+}
```

Comment:
Shall we use `start` and `end` instead of `left` and `right`?

Reply:
Maybe we need something similar to the current `TopAppBarSection` component that receives an `align` prop. 

The new component will be used together with `HeaderBar`:

```tsx
<HeaderBar>
    <HeaderBar.Section align="start">Left component</HeaderBar.Section>
    <HeaderBar.Section>Center component</HeaderBar.Section>
    <HeaderBar.Section align="end">Right component</HeaderBar.Section>
</HeaderBar>
```

Reply:
Awesome, making this change now!